Wrappers/Python/ccpi/viewer/utils/io.py

```diff
@@ -0,0 +1,45 @@
+
+import glob
+import os
+import re
+
+import vtk
+
+
+def SaveRenderToPNG(render_window, filename):
+    ''' Saves contents of a vtk render window
+    to a PNG file.
+
+    Parameters
+    ----------
+    render_window: vtkRenderWindow
+        render window to save contents of.
+    filename: str
+        name of file to write PNG to.
+    '''
+    w2if = vtk.vtkWindowToImageFilter()
+    w2if.SetInput(render_window)
+    w2if.Update()
+
+    basename = os.path.splitext(os.path.basename(filename))[0]
+    # Note regex matching is different to glob matching:
```

Comment:
would that be a problem?

Reply:
it's not a problem it just explains why there are both the `regex` and the `fname_string` variables which look very similar